src/server/types.rs

```diff
@@ -231,8 +231,10 @@ impl StatusInfo {
 pub enum StatusType {
     #[default]
     Undeployed,
-    Compensating,
-    Ready,
+    #[serde(alias = "Compensating")]
```

Comment:
Does `alias` "stack" with `rename_all = "lowercase"`? I'm curious if this should be `compensating`?

Reply:
Good question. Let me see if I can test that